content/blog/reflections-on-excalidraw/index.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Reflections on Excalidraw
 date: 2020-01-15
+note: 'This post appeared first on <a href="https://blog.vjeux.com/2020/uncategorized/reflections-on-excalidraw.html">Vjeux’s blog</a>.'
```

Comment:
cc @vjeux Is this ok or should it be “Christopher Chedeau’s blog?”

Reply:
I prefer to be called Vjeux on the internet. This is good :)